react/wbooks/src/app/screens/Home/styles.module.scss

```diff
@@ -4,13 +4,14 @@
 
 .books-container {
   flex-wrap: wrap;
-  margin-left: auto;
-  margin-right: auto;
-  margin-top: $navbar-height;
+  margin: $navbar-height auto 0 auto;
```

Comment:
is there a way to make the margin in Navbar component? iin order to use margin-bottom and generalize this margin

Reply:
This margin is to separate the books container from the top of screen (top 0). The margin is not compared to the navbar I think because the navbar has position fixed